utils/check_metadata.go

```diff
@@ -3,7 +3,7 @@ import (
 	"github.com/yunify/qingstor-sdk-go/v3/request/errors"
```

Comment:
imports should be organized as following:

```golang
systemd

3rd

self
```

for example:

```golang
"io"
"os"

"github.com/gooogle/grpc"

"github.com/yunify/qingstor-sdk-go/v3/request/errors"
```

Keep the same order makes it easier for reading.